dipper/curie_map.yaml

```diff
@@ -1,3 +1,7 @@
+# Skolemize Blank Nodes
+# overwrite iri fragment with '' for unresovable bnodes
+'_':    'https://monarchinitiative.org/.well-known/genid/'
```

Comment:
> is there a way in owl to declare a node as anonymous

Not as such. If your axiom is about an IRI, it's about an IRI. If it's about a blank node, it's about a blank node. 

Of course we can create our own annotation property called say 'pseudo-anonymous', and use that to annotate any IRI.